src/containers/repository_issue.py

```diff
@@ -0,0 +1,130 @@
+import re
+from typing import List, Optional
+
+from .milestone import Milestone
+
+
+class RepositoryIssue:
+    def __init__(self):
+        self.number: int = 0
+        self.organization_name: str = ""
+        self.repository_name: str = ""
+        self.title: str = ""
+        self.state: str = ""
+        self.url: str = ""
+        self.body: str = ""
+        self.created_at: str = ""
+        self.updated_at: str = ""
+        self.closed_at: Optional[str] = None
+        self.milestone: Optional[Milestone] = None
+        self.labels: List[str] = []
+        self.page_filename: str = ""
+
+    def load_from_json(self, issue_json, repository):
+        # TODO: is possible to remove load and save from and to json, so it is json native
+        self.number = issue_json["number"]
+        self.organization_name = repository.organization_name
+        self.repository_name = repository.repository_name
+        self.title = issue_json["title"]
+        self.state = issue_json["state"]
+        self.url = issue_json["html_url"]
+        self.body = issue_json["body"]
+        self.created_at = issue_json["created_at"]
+        self.updated_at = issue_json["updated_at"]
+        self.closed_at = issue_json["closed_at"]
+
+        # Have to initialize milestone before loading from JSON, so it has default values
+        milestone_json = issue_json['milestone']
+        self.milestone = Milestone()
+
+        if milestone_json is not None:
+            self.milestone.load_from_api_json(milestone_json)
+
+        labels = issue_json.get('labels', [])
+        self.labels = [label['name'] for label in labels]
+
+        md_filename_base = f"{self.number}_{self.title.lower()}.md"
+        self.page_filename = self.sanitize_filename(md_filename_base)
+
+    def to_dict(self):
+        return {
+            "number": self.number,
+            "organization_name": self.organization_name,
+            "repository_name": self.repository_name,
+            "title": self.title,
+            "state": self.state,
+            "url": self.url,
+            "body": self.body,
+            "created_at": self.created_at,
+            "updated_at": self.updated_at,
+            "closed_at": self.closed_at,
+            "milestone_number": self.milestone.number,
+            "milestone_title": self.milestone.title,
+            "milestone_html_url": self.milestone.html_url,
+            "labels": [str(label) for label in self.labels],
+            "page_filename": self.page_filename
+        }
+
+    def filter_out_labels_in_description(self, label_name: str, repository_issues: List['RepositoryIssue']):
```

Comment:
This method is not working with issue's description/body

Reply:
This method is not supposed to work with issue's body. The method only checks, if the label is really mentioned in the label section. Because GH filter looks for label_name appearance in the whole issue content.

Reply:
Solved by using different way of fetching repository issues.